examples/hypar_example.py

```diff
@@ -0,0 +1,218 @@
+
+        print('Joining sentences')
+        review['sentence'] = review['sentence'].apply(lambda x: x.replace('\t', ' '))
+
+        print('Saving reviews')
+        review.to_csv(re_path, index=False, header=False, sep='\t')
+        del review
+    else:
+        print('Warning: review.txt already exists, skipping.')
+
+    df.drop(columns=['sentence'], inplace=True)  # space efficiency
+
+    # Creating sentiment file
+    s_path = os.path.join('seer-ijcai2020', dataset_name, 'sentiment.txt')
+    if not os.path.isfile(s_path):
+        print('Joining aspects, opinions and sentiments')
+        df['aspect:opinion:sentiment'] = df.apply(lambda x: f"{x['aspect']}:{x['opinion']}:{x['sentiment']}", axis=1)
+        df.drop(['aspect', 'opinion', 'sentiment'], axis=1, inplace=True)
+
+        print('Assigning aspects, opinions and sentiments to reviews')
+        aos = df.groupby(['reviewerID', 'asin'], as_index=False).agg({'aspect:opinion:sentiment': ','.join})
+
+        print('Saving sentiment')
+        with open(s_path, 'w') as f:
+            for _, (r, a, t) in aos.iterrows():
+                f.write(f'{r},{a},{t}\n')
+    else:
+        print('Warning: sentiment.txt already exists, skipping.')
+
+
+def load_feedback(fmt="UIR", reader: Reader = None) -> List:
+    """Load the user-item ratings, scale: [1,5]
+
+    Parameters
+    ----------
+    reader: `obj:cornac.data.Reader`, default: None
+        Reader object used to read the data.
+
+    Returns
+    -------
+    data: array-like
+        Data in the form of a list of tuples (user, item, rating).
+    """
+    fpath = 'seer-ijcai2020/cellphone/ratings.txt'
+    reader = Reader() if reader is None else reader
+    return reader.read(fpath, fmt=fmt, sep=",")
+
+
+def load_review(reader: Reader = None) -> List:
+    """Load the user-item-review list
+
+    Parameters
+    ----------
+    reader: `obj:cornac.data.Reader`, default: None
+        Reader object used to read the data.
+
+    Returns
+    -------
+    data: array-like
+        Data in the form of a list of tuples (user, item, review).
+    """
+    fpath = 'seer-ijcai2020/cellphone/review.txt'
+    reader = Reader() if reader is None else reader
+    return reader.read(fpath, fmt="UIReview", sep="\t")
+
+
+def load_sentiment(reader: Reader = None) -> List:
+    """Load the user-item-sentiments
+    The dataset was constructed by the method described in the reference paper.
+
+    Parameters
+    ----------
+    reader: `obj:cornac.data.Reader`, default: None
+        Reader object used to read the data.
+
+    Returns
+    -------
+    data: array-like
+        Data in the form of a list of tuples (user, item, [(aspect, opinion, sentiment), (aspect, opinion, sentiment), ...]).
+
+    References
+    ----------
+    Gao, J., Wang, X., Wang, Y., & Xie, X. (2019). Explainable Recommendation Through Attentive Multi-View Learning. AAAI.
```

Comment:
This sentiment reference should refer back to this [paper](https://www.yongfeng.me/attach/efm-zhang.pdf) instead. Because you are using data extracted from Sentires toolkit.
Or you can just simply refer to this repo https://github.com/evison/Sentires